src/content_script.js

```diff
@@ -40,6 +40,8 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
   "[aria-label*='Log in with Facebook']", // Tinder
   "[action*='facebook_login']", // Airbnb
   "[action*='facebook_signup']", // Airbnb
+  "button[color='facebookBlue']", // Patreon
+  "[class*='btn-facebook']", // Mixcloud
```

Comment:
This selector is too broad of a class and will identify non-login items accidentally. Recommend adding some parent classes, `.modal-content`, etc.